armada/handlers/armada.py

```diff
@@ -66,8 +66,8 @@ def __init__(self,
         self.wait = wait
         self.timeout = timeout
         self.tiller = Tiller(tiller_host=tiller_host, tiller_port=tiller_port)
-        self.documents = yaml.safe_load_all(file)
-        self.config = self.get_armada_manifest()
+        self.documents = list(yaml.safe_load_all(file))
```

Comment:
when we use a yaml.load it creates an object that can be lazy iterated on and its kinda a one off, we need to cast it to a list so we can make it iterate. .